### PR TITLE
[FEATURE] Redesign issues by new database

### PR DESCRIPTION
**Pull request by Markmax2304**
Notice. authorId and executorId doesn't still work properly. And if you want to create new issue, you must have at least one user in database with id equals 1

### CODE REVIEW

File: src/main/java/app/Util/Path.java [hunk 1]
```diff
@@ -2,18 +2,6 @@
 
 public class Path
 {
-    public static class Web {
```
Comment (Ruslanwww) [inline src/main/java/app/Util/Path.java]: return back for reuse

File: src/main/java/app/Issue/IssueController.java [hunk 2]
```diff
@@ -1,81 +1,159 @@
...
+import java.util.List;
 import java.util.Map;
 
-import static app.Application.*;
 import static app.Util.RequestUtil.*;
 
 public class IssueController
 {
     public static Handler fetchAllIssues = ctx -> {
+        Base.open();
```
[the remark this answers is not in the document]
Comment (Markmax2304) [reply src/main/java/app/Issue/IssueController.java]: I have said about it.
If remove this statement accessing to database throws exception

File: src/main/java/app/Issue/IssueController.java [hunk 3]
```diff
@@ -1,81 +1,159 @@
...
         Map<String, Object> model = ViewUtil.getBaseModel(ctx);
-        model.put("issues", issueDao.getAllIssues());
+
+        List<Issue> issues = Issue.findAll();
+        model.put("issues", issues);
+
         ctx.render(Path.Template.ISSUES, model);
+        Base.close();
```
[the remark this answers is not in the document]
Comment (Markmax2304) [reply src/main/java/app/Issue/IssueController.java]: the same above

File: src/main/java/app/Util/Route.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import app.Issue.IssueController;
+import io.javalin.apibuilder.EndpointGroup;
+
+import static io.javalin.apibuilder.ApiBuilder.get;
+import static io.javalin.apibuilder.ApiBuilder.post;
+
+public class Route
+{
+    public static EndpointGroup routes = () ->  {
+        get("/", IssueController.fetchAllIssues);       // just test
```
Comment (Ruslanwww) [inline src/main/java/app/Util/Route.java]: add
`before(ctx -> Base.open());`
`after(ctx -> Base.close());`
and test it

**Pull request merged**